samples/charts/data-chart/shape-series/Pages/DataChartRangeSeries.razor

```diff
@@ -0,0 +1,135 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
+<div class="igContainer">
+
+    <div class="igOptions-horizontal">
+        <span class="igOptions-label">Type of Shape Series: </span>
+        <select value=@SeriesType
+                @onchange="OnSeriesTypeChanged">
+            <option>Polygon</option>
+            <option>Polyline</option>
+        </select>
+
+        <span class="igLegend-title">Legend: </span>
+        <div class="igLegend">
+            <Legend @ref="LegendRef" Orientation="LegendOrientation.Horizontal" />
+        </div>
+    </div>
+
+    <div class="igComponent">
+        @if (DataSource != null)
+        {
```

Comment:
there should be 2 @if (SeriesType  == "ScatterPolygonSeries" ) conditions each with one type of series (see StackedSeries sample):
ScatterPolygonSeries, ScatterPolylineSeries,